### ISSUE TITLE
Module seed doesn't work [L5.3]

### ISSUE THREAD

**Issue by rifkyaziz** [caffeinated/modules#245]
Hay i have problem with module:seed artisan command. when i try to run module:seed, it give me nothing, no error warning or anything else. i also run composer dump-autoload and php artisan optimize. but still doesnt work. 
this is my HregistrationDatabaseSeeder(Hregistration is my module):

```
<?php

use Illuminate\Database\Seeder;

class HregistrationDatabaseSeeder extends Seeder
{
    /**
     * Run the database seeds.
     *
     * @return void
     */
    public function run()
    {
        $this->call(ProvinsiTableSeeder::class);
        $this->call(KabupatenTableSeeder::class);
        $this->call(KecamatanTableSeeder::class);
        $this->call(KelurahanTableSeeder::class);
    }
}
```

**Comment by nurmanhabib** [comment]
This is because the generated seeder in modules do not include namespace. In addition to the console `ModuleSeedCommand` requires manifest `namespace`, whereas in the cache is only provided key `basename` in `storage/app/modules.json`

It seems to have not found a solution, I'm looking for an alternative as well. A temporary solution is to make the seeder at the root app with `php artisan make:seeder <ClassName>`

**Comment by nurmanhabib** [comment]
-_- tiwas nggo b inggris, jebul si rifky ( in javanese :smile: )

**Comment by A-Lawrence** [comment]
Fixed in PR #272.

@rifkyaziz @nurmanhabib Until @kaidesu accepts the PR, you can (although it's not recommend) make the changes from the PR yourself.
